<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDiskDialog</class>
 <widget class="QDialog" name="NewDiskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Disk</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lNewDiskName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name for new disk:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNewDiskName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editLocation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lCatPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Location to catalogue:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseLocation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lCatalogue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new disk to catalogue:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editNewDiskName</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>editLocation</tabstop>
  <tabstop>chooseLocation</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewDiskDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewDiskDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
